### DISCUSSION TITLE
Faster Plotting after 1.0.3 update

### DISCUSSION THREAD

**Discussion by Teledyol** [Chia-Network/chia-blockchain#1645] [General]
Anyone else seeing this?

Just using the GUI on macOS 11.2.3 • Mac Pro 2013 • 3.5 GHz 6-Core Intel Xeon E5 •128 GB 1066 MHz DDR3 • AMD FirePro D300 2 GB • Aura 2T SSD • 32T Final at 40 plots and counting

No error messages after 6:30p PST last night…

Plotting four in parallel, a couple more days will tell.

**Comment by dorkmo** [comment]
they sped up bitfield plotting, mentioned in the release notes of 1.0.2
"This includes a 20% speed increase for bitfield plotting compared to the previous version on the same machine. In many cases this will mean that bitfield plotting is as fast or faster than non bitfield plotting."
keep at it!

**Reply by Teledyol** [reply]
In the following attempt to plot 4 in parallel, two plots stuck at 79% and 73%. I'm just leaving them in place, which may be a mistake. 

I started a new task of running 4 queues of 4 at the same time, to see if that works. At 5%, they seem to be working. We'll see…

**Reply by Teledyol** [reply]
Had to give up on those two plots when my available space fell below 4G on a 2T SSD.

I started dumping the .tmp file then deleting the two plots on GUI. It appears to have worked in releasing space without disrupting the four tasks running

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
